Managers/LuaMan.cpp

```diff
@@ -869,7 +869,9 @@ int LuaMan::Initialize() {
 			.def("RemoveEmitter", (bool (MOSRotating::*)(Attachable *attachableToRemove))&MOSRotating::RemoveAttachable)
 			.def("RemoveEmitter", (bool (MOSRotating::*)(long uniqueIDOfAttachableToRemove))&MOSRotating::RemoveAttachable)
 			.def_readonly("Attachables", &MOSRotating::m_Attachables, return_stl_iterator)
-			.def_readonly("Wounds", &MOSRotating::m_Wounds, return_stl_iterator),
+			.def_readonly("Wounds", &MOSRotating::m_Wounds, return_stl_iterator)
+			.property("GibSound", &MOSRotating::GetGibSound, &MOSRotating::SetGibSound, detail::null_type(), adopt(_2)),
```

Comment:
It's not a big deal, and don't bother with it if you don't want to, but it's nice to try to match up where these luaman definitions sit with where the variables sit, and also to try to group properties roughly together.

Like I said though, don't worry about it if you don't wanna deal with it, this is okay as-is.